README.md

```diff
@@ -43,6 +43,12 @@ Ncurses and tiles versions are available in the [official repos](https://tracker
 
 `sudo apt install cataclysm-dda-curses cataclysm-dda-sdl`
 
+#### Flathub
+
+<a href="https://flathub.org/apps/org.cataclysmdda.CataclysmDDA">
+    <img src="https://flathub.org/assets/badges/flathub-badge-en.png" alt="Download from Flathub" width="200"/>
+</a>
```

Comment:
The preview at least is refusing to render this, how about just a text link?

Reply:
ok, i can make a simple text link